api/clients/accountant_test.go

```diff
@@ -271,7 +272,7 @@ func TestConcurrentBinRotationAndAccountBlob(t *testing.T) {
 			defer wg.Done()
 			// for j := 0; j < 5; j++ {
 			// fmt.Println("request ", i)
-			_, err := accountant.AccountBlob(ctx, 100, quorums)
+			_, err := accountant.AccountBlob(ctx, 100, quorums, salt)
 			assert.NoError(t, err)
 			time.Sleep(500 * time.Millisecond)
```

Comment:
Do we need this sleep? Sleep in tests aren't reliable/guaranteed

Reply:
I wanted to simulate a consistent dispersal scenario across multiple reservation periods. Would this sit better with inabox tests?

Reply:
If each call is made 500ms apart, are these calls actually concurrent? 
One way to remove sleep would be to take `now` as a parameter in `AccountBlob` instead of setting `now := time.Now().Unix()` in the method itself.

Reply:
You are right, I removed this sleep and the tests still passes:)